test_harness/slacker.py

```diff
@@ -8,9 +8,11 @@ class Slacker:
     def __init__(self, url=None):
         self.url = url if url else os.getenv("SLACK_WEBHOOK_URL")
 
-    async def post_notification(self, messages=[]):
+    async def post_notification(self, messages=None):
```

Comment:
Just for my understanding, what's the advantage of doing it this way? I guess if messages is passed in as None?

Reply:
PyCharms complained that the "Default argument value is mutable". I guess replacing it with a Tuple removes this complain?